src/App.js

```diff
@@ -0,0 +1,70 @@
+import { useState } from "react";
+import "./App.css";
+import Cart from "./components/Cart";
+import Navbar from "./components/Navbar";
+import ProductsMenu from "./components/ProductsMenu";
+import Checkout from "./components/Checkout";
+
+const emptycart = [];
+
+function App() {
+  const [cart, setCart] = useState(emptycart);
+  const [fullname, setFullname] = useState("");
+  const [email, setEmail] = useState("");
+
+  const handleSubmit = () => {
+    console.log("hi", cart);
+    setCart(emptycart);
+    setFullname("");
+    setEmail("");
+  };
+  const handleDelete = ({ id }) => {
+    setCart((prevcart) => {
+      return prevcart.filter((product) => product.id !== id);
+    });
+  };
+  const totalprice = cart.reduce(
+    (total, product) => total + product.price * product.count,
+    0
+  );
+
+  const HandleSuccessfullAdd = ({ id, title, price, image, rating, count }) => {
+    const product = {
```

Comment:
you are destructing and then constructing an object again 🤔

Reply:
done it